src/Base/PlacementPy.xml

```diff
@@ -63,14 +63,19 @@ Alias to move(), to be compatible with TopoShape.translate().\n
 vector : Base.Vector\n    Vector by which to move the placement.</UserDocu>
 			</Documentation>
 		</Methode>
-        <Methode Name="rotate">
+        <Methode Name="rotate" Keyword="true">
         <Documentation>
-            <UserDocu>rotate(center, axis, angle) -> None\n
+            <UserDocu>rotate(center, axis, angle, kwargs) -> None\n

```

Comment:
Shouldn't `kwargs` be `comp` here?

Reply:
I'm no Python expert and not sure about that, but I believe that's how keyword arguments are commonly documented. With the introduction of keyword arguments, all arguments can be given by keyword, not just comp. Only positional arguments, if given, have to be specified in the order shown. So you can write

    rotate(center, angle=45, axis=(0,0,1), comp=False)
or

    rotate(center, axis, angle)

but not

    rotate(center, axis, angle, True)

Reply:
OK, thanks for explaining (I am not a Python expert either).

Reply:
`**kwargs` is when you have an unknown number of keyword arguments. `*args` when you have an unknown number of non-keyword arguments. Since `comp` is a known argument with a default value, it should just be `comp` in the argument list.

Reply:
`**kwargs` is when you have an unknown number of keyword arguments. `*args` when you have an unknown number of non-keyword arguments. Since `comp` is a known argument with a default value, it should just be `comp` in the argument list. @Roy-043 @mfro0

Reply:
@chennes